tests/TestCase/View/Helper/FormHelperTest.php

```diff
@@ -1742,24 +1745,24 @@ public function testFormSecurityMultipleControlDisabledFields()
         $result = $this->Form->secure($this->Form->fields);
         $hash = '43c4db25e4162c5e4edd9dea51f5f9d9d92215ec%3AAddresses.0.id%7CAddresses.1.id';
         $tokenDebug = urlencode(json_encode([
-                '/articles/add',
```

Comment:
Please avoid side effects of indentation changes, only minimal diffs, please.
Also, these are technically now incorrect, as the normalized form would have 2 levels here.